sdk/eventhub/azure-eventhub/azure/eventhub/_retry.py

```diff
@@ -0,0 +1,11 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from typing import Optional, Dict, Any
+
+from enum import Enum
+
+class RetryMode(str, Enum):
+    Exponential = 'exponential'
+    Fixed = 'fixed'
```

Comment:
as per the guideline: DO use UPPERCASE names for enum names. https://azure.github.io/azure-sdk/python_design.html#enumerations
```suggestion
    EXPONENTIAL = 'exponential'
    FIXED = 'fixed'
```

Reply:
🤦 thanks!